test/TestAssets/QualityToolsAssets/Microsoft.TestPlatform.QTools.Assets.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is there any specific reason for creating nuget package. Cant we use the Webtest directly as we are using other test assets like `SampleTestProject.csproj`

Reply:
webtest is there in enterprise edition. you cant build in ur repo